ui/pages/marketplace/index.tsx

```diff
@@ -70,27 +68,7 @@ export default function Marketplace({ listings }: MarketplaceProps) {
         <ContentLayout
           header="Marketplace"
           headerSize="max(20px, 3vw)"
-          description={
```

Comment:
Where is this moved to?

Reply:
Line 52 of that file is where I define the description component